lib/buildpackutil.py

```diff
@@ -304,16 +314,24 @@ def lazy_remove_file(filename):
             raise
 
 
-def ensure_and_get_mono(directory, cache_dir):
+def ensure_and_get_mono(directory, cache_dir, mendix_version):
     try:
-        return _get_mono_path(directory)
+        return _get_mono_path(directory, mendix_version)
     except NotFoundException:
-        download_and_unpack(
-            get_blobstore_url('/mx-buildpack/mono-3.10.0.tar.gz'),
-            directory,
-            cache_dir,
-        )
-    return _get_mono_path(directory)
+        if LooseVersion(mendix_version) < LooseVersion('6.8'):
```

Comment:
define a function `def _get_mono_version(mendix_version)` and use that because there's code duplication